modules/escalation/README.md

```diff
@@ -0,0 +1,50 @@
+## Escalation
+
+Terraform module to configure [Opsgenie Escalation](https://registry.terraform.io/providers/opsgenie/opsgenie/latest/docs/resources/escalation)
+
+
+## Usage
+
+[Create Opsgenie Escalation example](../../examples/escalation)
+
+```hcl
+module "escalation" {
+  source = "git::https://github.com/cloudposse/terraform-opsgenie-incident-management.git//modules/escalation?ref=master"
+
+  provider_api_key = var.opsgenie_provider_api_key
+
+  escalation = {
+    name          = module.label.id
+    owner_team_id = module.owner_team.team_id
+
+    rule = {
+      recipients = [{
+        type = "team"
+        id   = module.escalation_team.team_id
+      }]
+    }
+  }
+
+}
+
+```
+
+## Inputs
+
+**Note:** `escalation` is a map for two reasons: 
+- to be able to put whole configuration in yaml file
+- variables defined with type set are not robust enough (can't set default values)
+
```

Comment:
Hrmmmmmm this is where the submodules cause scope creep. Our readme generation/validation stuff won't work. I know there's recommit stuff for that. Just it's another convention.

Reply:
What do you mean exactly? Do you mean using map for whole escalation or are you referring to readme generation from yaml?

> I know there's recommit stuff for that.
Can you elaborate?

Reply:
👍